libs/remix-ui/solidity-uml-gen/src/lib/solidity-uml-gen.tsx

```diff
@@ -8,8 +8,22 @@ export interface RemixUiSolidityUmlGenProps {
   updatedSvg?: string
   loading?: boolean
   themeSelected?: string
+  themeName: string
 }
 
+const themeCollection = [
```

Comment:
`themeCollection` is a duplicate of the theme config in solidity-umlgen.tsx. It may be better to manage seetings from a single source, probably in a separate file.

Reply:
True, thanks @ioedeveloper